packages/components/src/focal-point-picker/index.js

```diff
@@ -259,10 +257,7 @@ export class FocalPointPicker extends Component {
 		} = this.state;
 
 		if ( bounds.left === undefined || bounds.top === undefined ) {
-			return {
-				left: '50%',
-				top: '50%',
-			};
```

Comment:
Just to remove redundancy because the only place `iconCoordinates` was called was to create the `coordinates` prop for the sub-component `FocalPoint` which defaults it to the same:
https://github.com/WordPress/gutenberg/blob/a9e8a8565e2c7bdf3e44a04462d96db5288428ff/packages/components/src/focal-point-picker/focal-point.js#L16-L17